src/overlays/actors/ovl_En_Test4/z_en_test4.h

```diff
@@ -9,10 +9,20 @@ typedef void (*EnTest4ActionFunc)(struct EnTest4*, GlobalContext*);
 
 typedef struct EnTest4 {
     /* 0x0000 */ Actor actor;
-    /* 0x0144 */ char unk_144[0xC];
+    /* 0x0144 */ s8 unk_144; // 0 on night, 1 on day
```

Comment:
Since we know how it works, could this be named? Maybe `isDay`? I'm not sure it matches if you treat it like a boolean, so maybe a different name is better.

Reply:
Actually, I have it named `isDay` before, but it is used as an array index, so I though it was correct to name it like that.

Reply:
Ah, you're right. I guess it's not actually used as a boolean in all cases then.